### ISSUE TITLE
Braintree TLSv1.2 supported SDKs

### ISSUE THREAD

**Issue by VedranBrnjetic** [nopSolutions/braintree-plugin-for-nopcommerce#2]
Hi, 

A notification just appeared on my Braintree Payment gateway informing me that they will be dropping TLS 1.0 and 1.1 support in production:

"As part of PCI 3.1 requirements, we will be deprecating support for TLS 1.0 and 1.1 in Production on June 30, 2017. You can find information on how to update your systems here - http://bit.ly/2h5v0w3. "

I hope this is the right place to poke about it as I am using this plugin on my website. Can you tell me what it means for me?

**Comment by AndreiMaz** [comment]
@skoshelev Назначаю тебе. посмотри

**Comment by skoshelev** [comment]
@AndreiMaz @VedranBrnjetic 
We use the official version of the SDK for Braintree version 3.4, from the version 3.1 the SDK supports TLS v1.2. I have found this code in their source code from [GitHub](https://github.com/braintree/braintree_dotnet/blob/26cc4d590459d08440e27a378dcf291a2a58ef66/src/Braintree/BraintreeService.cs):

```
const int SecurityProtocolTypeTls12 = 3072;
ServicePointManager.SecurityProtocol = ServicePointManager.SecurityProtocol | ((SecurityProtocolType)SecurityProtocolTypeTls12);
```

**Comment by skoshelev** [comment]
Closed #2
